db/schema.rb

```diff
@@ -43,8 +43,6 @@
     t.boolean  "fogged"
   end
 
-  add_index "checkins", ["device_id"], name: "index_checkins_on_device_id", using: :btree
```

Comment:
Don't remember changing that, I'll try rake db:migrate and see if it goes away but I remember migrating before pushing the branch though